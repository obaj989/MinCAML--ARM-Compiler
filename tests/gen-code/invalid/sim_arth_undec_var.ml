let x = 2 in
let y = u in
x + y
