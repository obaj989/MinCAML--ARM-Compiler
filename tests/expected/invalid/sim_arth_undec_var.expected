[31m Error! Undeclared variable u is being used. ![0m
